backend/apps/recommendations/tasks.py

```
@@ -11,3 +16,46 @@ def generate_user_recommendations(user_id):
+                        issue_number=issue["number"],
+                        defaults={
+                            "title": issue["title"],
+                            "url": issue["html_url"],
+                            "labels": [label["name"] for label in issue.get("labels", [])],
+                            "difficulty": "beginner",
+                            "is_open": True,
+                        }
+                    )
+            else:
+                logger.warning(f"Failed to fetch issues for {repo}: {response.status_code}")
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)